### ISSUE TITLE
Accessibility for disabled dropdown menu: flagged as not accessible for low vision users

### ISSUE THREAD

**Issue by ktitsworth** [carbon-design-system/carbon#558]
-->

## Detailed description

The Identity and Access team for IBM Cloud received an accessibility flag for using the disabled but filled in state for the Carbon dropdown. This means the dropdown in the disabled state did not have sufficient contrast for a low vision user to read it without additional assistance software. https://github.ibm.com/IAM/access-management/issues/4270

> Is this issue related to a specific component? 

Dropdown, disabled.

> What did you expect to happen? What happened instead? What would you like to see changed?

Expected it to be accessibility compliant.

> What browser are you working in? 

See original ticket: https://github.ibm.com/IAM/access-management/issues/4270

> What version of the Carbon Design System are you using? 

Current

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM Cloud

## Steps to reproduce the issue

 1. Go to https://dev-console.stage1.ng.bluemix.net/
 2. Login with IBM id
 3. Click Manage --> Security
 4. Click Identity & access -> Service IDs
 5. Click one created Service ID
 6. Click Edit policy in menu list from one policy

**Comment by marijohannessen** [comment]
@aagonzales what are the rules when it comes to disabled styles and accessibility?

**Comment by aagonzales** [comment]
So my understanding is that since the disabled state is un-useable by everyone it does not have to pass color contrast accessibility. In fact, a screen reader would completely skip over a disabled field all together. 

*https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html
Section 1.4.1

> **Disabled Form elements.**
> Form elements which are disabled via markup or scripting, are greyed out and made inactive by the user agent. When in the disabled state these elements do not receive focus. Assistive technologies can programmatically determine the state of disabled elements and will provide this information to the user as the elements are encountered on the page. The change in color and loss of focus provides redundant, visual information about the state of the control.

**Comment by aagonzales** [comment]
Further research says that if a field still needs to be read but not accessed then it should be given a "read only" state. This read only state would have to pass color contrast and it would still be read by a screen reader (and not passed over like a disabled state). We do not currently have "read only" states but if this is more of the situation that you're encountering than we can look into it. 

@ktitsworth I tired to replicate the steps to see the problem/scenario but I never saw where the disabled dropdown was. Could you please screenshot the page for context?

**Comment by nicoxxie** [comment]
@aagonzales I uploaded an image see https://github.ibm.com/IAM/access-management/issues/4270#issuecomment-4994254

**Comment by aagonzales** [comment]
@nicoxxie So what I'm seeing doesn't look like what we recommend for disabled dropdown should be. Along with the entry the label and any icons with in it should also be disabled. Is this design trying to say on item inside of the dropdown is disabled? If that's the case why are you showing it at all if they can't select it?

Also referring to my above comments any element that is "disabled" does NOT have to pass color contrast according to the official accessibility rules.

**Comment by tay-aitken** [comment]
@nicoxxie is this still an active issue? If no response in 3 days this issue will be closed.

**Comment by tay-aitken** [comment]
Closing issue due to inactivity. Feel free to re-open if this is still an issue.
